week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -1,98 +1,233 @@
+ * Look for usage of the function in the code and see what variable is passed into it as an argument.
+ * 
+ * Create and insert the below HTML elements to the DOM for each of Objects in the Array as follows:
+ *  1. Add a <h1> tag to the website containing the name of the person
+ *  2. Add a <h2> tag to the website containing the job of the person
  *
- * For each of the names in the array passed into this function
- *
- * - Add a <h1> tag to the website containing the name of the person
- * - Add a <h2> tag to the website containing the job of the person
- *
- * All of your HTML should go inside the Div tag with the id "content".
- *
+ * All of your HTML elements should go inside the Div tag with the id "content".
+ * 
+ * An example "content" div should look like:
  * <div id="content">
- *      <h1>{Name Here}</h1>
- *      <h2>{Job Here}</h2>
- *      .....
+ *      <h1>Mario</h1>
+ *      <h2>Plumber</h2>
+ *      <h1>Luigi</h1>
+ *      <h2>Plumber</h2>
  * </div>
  */
-function exerciseOne(arrayOfPeople) {
-  let content = document.querySelector("#content");
-}
+// let people = [
+//   { name: "Chris", job: "Teacher" },
+//   { name: "Joanna", job: "Student" },
+//   { name: "Boris", job: "Prime Minister" }
+// ];
+function insertPeopleData(people) {
+  var arrayLength = people.length;
+  if (arrayLength > 0) {
+
+    for (let i in people) {
+  
+      //console.log(people.name[i]);
+      var element1 = document.createElement("h1");
+      
+      var t1 = document.createTextNode(people[i].name);
+      element1.appendChild(t1);
+      document.body.appendChild(element1);
+   
+      var element3 = document.createElement("h2");
+      var t3 = document.createTextNode(people[i].job);
+      element1.appendChild(t3);
+      document.body.appendChild(element3);
+    }
+   
+  }
+  
+  }
+
+
+//   //Write your code in here
+// }
 
 /**
  *
- * Create a list of shopping items. You should use an unordered list.
- *
- * All of your HTML should go inside the Div tag with the id "content".
+ * Create a list of shopping items using an unordered HTML list.
+ * The input of this function is a simple Array of Strings, look for the concrete example in the code.
+ * 
+ * All of your HTML elements should go inside the Div tag with the id "content".
  *
+ * Hint for type of lists in HTML: https://www.w3schools.com/html/html_lists.asp
  */
-function exerciseTwo(shopping) {
+let myShoppingList = ['Pizza', 'Pasta', 'Milk', 'Meat', 'Chicken', 'Mangos', 'Strawberries'];
+//get the container
+
+function insertShoppingList(myShoppingList) {
+  //get the container 
+  let container = document.getElementById('content');
+  //calculate the length of the array.
+  let arrayLength = myShoppingList.length;
+  if (arrayLength > 0) {
+    //create an unorder list if element present in array.
+    var myList = document.createElement('ul');
+    //add  class name to a list
+    myList.className = 'myClass';
```

Comment:
```suggestion
    myList.className = 'myClass';
```
Not sure what this class is for ?